include/vinecopulib/misc/tools_bobyqa.hpp

```diff
@@ -16,8 +16,8 @@ namespace vinecopulib {
 
 namespace tools_bobyqa {
 
-constexpr double sqrt_2 = std::sqrt(2.0);
-constexpr double sqrt_0_5 = std::sqrt(0.5);
+constexpr double sqrt_2 = 1.41421356237309504880168872420969807;
```

Comment:
Same as before, I think that we should use [boost](https://www.boost.org/doc/libs/1_37_0/libs/math/doc/sf_and_dist/html/math_toolkit/toolkit/internals1/constants.html)